packages/terra-date-picker/src/terra-dev-site/test/date-picker/DatePickerIgnoreInvalidSelectedDates.test.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/jsx-curly-brace-presence */
```

Comment:
why not fix the code?

Reply:
I didn't fix this portion because removing the curly braces creates another error in which the quotes can be escaped according to `eslintreact/no-unescaped-entities`.

Reply:
@mjhenkes https://github.com/cerner/terra-framework/pull/1164

Reply:
@emilyrohrbough I think that issue is relevant to the other override here: https://github.com/cerner/terra-framework/pull/1166/files#r445783234

I'd just fix this one by escaping the characters: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md

Reply:
Fixed the curly brace and escaping characters problem here: [d678858](https://github.com/cerner/terra-framework/pull/1166/commits/d67885871dee3d05aa83267625adaa0b9134ccc5).